compose/README.md

```diff
@@ -314,6 +314,20 @@ Optionally you may also want to delete the directories:
 
     $ rm -rf $HOME/.am/am-pipeline-data $HOME/.am/ss-location-data
 
+## Instrumentation: Running Prometheus and Grafana
+
+[Prometheus][instrumentation-0] and [Grafana][instrumentation-1] can be used
+to monitor Archivematica processes.
+
+To run them, reference the `docker-compose.instrumentation.yml` file:
+
+    $ docker-compose -f docker-compose.yml -f docker-compose.instrumentation.yml up -d
+
+Prometheus will start on localhost:9090; Grafana on localhost:3000.
```

Comment:
Could you make these actual links, please?

Reply:
Fixed.